.github/CODEOWNERS

```diff
@@ -6,6 +6,11 @@
 /avm/ptn/authorization/policy-assignment/ @Azure/avm-ptn-authorization-policyassignment-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/ptn/authorization/resource-role-assignment/ @Azure/avm-ptn-authorization-resourceroleassignment-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/ptn/authorization/role-assignment/ @Azure/avm-ptn-authorization-roleassignment-module-owners-bicep @Azure/avm-core-team-technical-bicep
+#/avm/ptn/avd-lza/insights/ @Azure/avm-ptn-avd-lza-insights-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
I take it this does not fail the static tests of any module?

Reply:
I just moved the line up. It's already existing and commented. What's your concern?

Reply:
Just asking if you ran the static tests of any module to test that nothing breaks. Just given that this file is checked by the static tests

Reply:
Tested and attached badge. I can run bulk static validation for all modules tomorrow

Reply:
No need. If one works, all work